lib/tapjoy/ldap/cli/audit/by_user.rb

```diff
@@ -0,0 +1,11 @@
+module Tapjoy::LDAP::CLI
+  module Audit
+    # Get a group to user mapping
+    class ByUser
+      def by_user
+        Tapjoy::LDAP::CLI::Audit.print_hash('Groups by user',
+          Tapjoy::LDAP::API::Audit.index_by_user)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Align the parameters of a method call if they span more than one line.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2768630346/issues/source?bid=3316970&fileBranchId=3362438#l7)